src/RealND.hpp

```diff
@@ -135,7 +137,7 @@ namespace espressopp {
       data[i] = v[i];
   }
   
-  inline RealND::RealND(const int _dim, const real v[3]) {
+  inline RealND::RealND(const int _dim, const real v[]) {
     setDimension(_dim);
```

Comment:
Temporarily I pushed a possible modification. Should write a test I think at some point for this in the spirit of the Array4DTest.